### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by BrubiSQL** [trimble-oss/dba-dash#204] [Ideas]
Hello :)

Implement some keyboard shortcuts in to DBA Dash, for example F5 to act as a window refresh and ESC to close any popup window we have open on to of the main one.

Regards,
B.

**Comment by DavidWiseman** [comment]
Thank you for the suggestion.

**Comment by DavidWiseman** [comment]
F5 to refresh has been added in 2.20.0.  
Alt+F4 can be used to close a window.

**Reply by BrubiSQL** [reply]
I shall endeavour to install the new version and test it asap! :)
Thank you for your work!
